### PR TITLE
Adding OAuth 1 support.

### PR DESCRIPTION
**Pull request by pavel-shelentsov**
https://smartbear.atlassian.net/browse/SOAPUIOS-194

### CODE REVIEW

**Comment by IlyaAvdeev** [comment]
@antone-sb  any objections?

File: soapui/src/main/java/com/eviware/soapui/support/editor/inspectors/auth/AuthorizationSelectionDialog.java [hunk 1]
```diff
@@ -99,10 +103,24 @@ private void createProfileForSelectedAuthType(XFormDialog dialog) {
...
-        if (!CredentialsConfig.AuthType.O_AUTH_2_0.equals(authType)) {
+            authTypeEnum = CredentialsConfig.AuthType.O_AUTH_2_0;
+        } else if (CredentialsConfig.AuthType.O_AUTH_1_0.toString().equals(authType)) {
+            profileName = dialog.getValue(AuthorizationTypeForm.OAUTH2_PROFILE_NAME_FIELD);
+            if (ProfileSelectionForm.isReservedProfileName(profileName)) {
+                UISupport.showErrorMessage("'" + profileName + "' is a reserved profile name.");
+                return;
+            }
+            if (getOAuth1ProfileContainer().getOAuth1ProfileNameList().contains(profileName)) {
+                UISupport.showErrorMessage("There is already a profile named '" + profileName + "'");
```
Comment (antone-sb) [inline soapui/src/main/java/com/eviware/soapui/support/editor/inspectors/auth/AuthorizationSelectionDialog.java]: Move to bundle.

File: soapui/src/main/java/com/eviware/soapui/support/editor/inspectors/auth/AuthorizationSelectionDialog.java [hunk 2]
```diff
@@ -99,10 +103,24 @@ private void createProfileForSelectedAuthType(XFormDialog dialog) {
...
                 return;
             }
+
             getOAuth2ProfileContainer().addNewOAuth2Profile(profileName);
-        }
-        CredentialsConfig.AuthType.Enum authTypeEnum = CredentialsConfig.AuthType.O_AUTH_2_0;
-        if (!CredentialsConfig.AuthType.O_AUTH_2_0.equals(authType)) {
+            authTypeEnum = CredentialsConfig.AuthType.O_AUTH_2_0;
+        } else if (CredentialsConfig.AuthType.O_AUTH_1_0.toString().equals(authType)) {
+            profileName = dialog.getValue(AuthorizationTypeForm.OAUTH2_PROFILE_NAME_FIELD);
```
Comment (antone-sb) [inline soapui/src/main/java/com/eviware/soapui/support/editor/inspectors/auth/AuthorizationSelectionDialog.java]: Maybe remove 2 from the field name? Looks confusing.

File: soapui/src/main/java/com/eviware/soapui/impl/rest/actions/oauth/GetOAuthAccessTokenAction.java [hunk 3]
```diff
@@ -40,7 +40,7 @@ public GetOAuthAccessTokenAction(OAuth2Profile target) {
     public void actionPerformed(ActionEvent event) {
         try {
             getOAuthClientFacade().requestAccessToken(target);
-        } catch (InvalidOAuth2ParametersException e) {
+        } catch (InvalidOAuthParametersException e) {
             UISupport.showErrorMessage("Invalid OAuth 2 parameters: " + e.getMessage());
```
Comment (pavel-shelentsov) [inline soapui/src/main/java/com/eviware/soapui/impl/rest/actions/oauth/GetOAuthAccessTokenAction.java]: All these string was moved to bundle.

File: soapui/src/main/java/com/eviware/soapui/impl/rest/actions/oauth/WebViewUserBrowserFacade.java [hunk 4]
```diff
@@ -33,6 +33,7 @@
 
     private WebViewBasedBrowserComponent browserComponent;
     private JFrame popupWindow;
+    private boolean modal = false;
```
Comment (pavel-shelentsov) [inline soapui/src/main/java/com/eviware/soapui/impl/rest/actions/oauth/WebViewUserBrowserFacade.java]: This option was deleted.

File: soapui/src/main/java/com/eviware/soapui/support/editor/inspectors/auth/AuthorizationSelectionDialog.java (hunk 2 above)
Comment (pavel-shelentsov) [inline soapui/src/main/java/com/eviware/soapui/support/editor/inspectors/auth/AuthorizationSelectionDialog.java]: Renamed.

**Pull request merged**